api/openapi-spec/swagger.json

```diff
@@ -17900,7 +17900,7 @@
      },
      "vmRolloutStrategy": {
       "description": "VMRolloutStrategy defines how changes to a VM object propagate to its VMI",
-      "$ref": "#/definitions/v1.VMRolloutStrategy"
+      "type": "string"
```

Comment:
9980250d683697402d5d13cf5be73639212f56ca should be amended to the relevant commit.